app/templates/index.html

```diff
@@ -2,7 +2,17 @@
 {% block title %}Home{% endblock %}
 {% block body %}
 <div class="jumbotron">
-  <h1>Flask Is Awesome</h1>
-  <p class="lead">And I'm glad to be learning so much about it!</p>
+    <h1>What is Genten Studios?</h1>
+    <p class="lead">Genten Studios is a group of programmers who are passionate about creating games. All of our games
```

Comment:
"group of programmers"
Hehem... And artists. May wanna say a group of creators instead so it also includes all other categories of contributors.

Reply:
Fixed